<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RTBiomanager</class>
 <widget class="QMainWindow" name="RTBiomanager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTBiomanager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="simulate">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>280</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameSynapse">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>581</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QStackedWidget" name="synapseModelPages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>561</width>
       <height>111</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageSynElec">
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>2</y>
         <width>261</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Model to external neuron conductance: </string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textSynElec_gMtoE">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>0</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>32</y>
         <width>261</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>External neuron to model conductance:</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textSynElec_gEtoM">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageSynGrad">
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>2</y>
         <width>261</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Model to external neuron conductances: </string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textSynGrad_gMtoE_slow">
       <property name="geometry">
        <rect>
         <x>333</x>
         <y>0</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textSynGrad_gMtoE_fast">
       <property name="geometry">
        <rect>
         <x>333</x>
         <y>30</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>62</y>
         <width>261</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>External neuron to model conductances:</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textSynGrad_gEtoM_slow">
       <property name="geometry">
        <rect>
         <x>333</x>
         <y>60</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textSynGrad_gEtoM_fast">
       <property name="geometry">
        <rect>
         <x>333</x>
         <y>90</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>295</x>
         <y>4</y>
         <width>61</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Slow</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>295</x>
         <y>33</y>
         <width>61</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Fast</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>295</x>
         <y>62</y>
         <width>61</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Slow</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>295</x>
         <y>91</y>
         <width>61</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Fast</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>48</y>
         <width>441</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency (kHz)</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textFreq">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textTime">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>70</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration (s)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkCalib">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>240</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibration</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkAnti">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>240</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Antiphase</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Channels (separated by commas)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>160</y>
      <width>61</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>200</y>
      <width>61</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textChannelInput">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>150</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textChannelOutput">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>190</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Neuron model</string>
    </property>
   </widget>
   <widget class="QComboBox" name="neuronModelCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Izhikevich</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hindmarsh-Rose</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rulkov Map</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Synapse model</string>
    </property>
   </widget>
   <widget class="QComboBox" name="synapseModelCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Electrical</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gradual chemical</string>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="frameNeuron">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>581</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QStackedWidget" name="neuronModelPages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>561</width>
       <height>121</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageIz">
      <widget class="QPlainTextEdit" name="textIzA">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textIzB">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textIzC">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textIzD">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textIzI">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>0</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textIzUIni">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textIzVIni">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>A</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>B</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>0</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>I</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>V Ini</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>U Ini</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageHr">
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X Ini</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textHrYIni">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textHrI">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>0</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>0</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>I</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>S</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textHrXIni">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>R</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textHrR">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textHrS">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Y Ini</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>90</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Z Ini</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textHrZIni">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>90</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageRlk">
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X Ini</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textRlkI">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>0</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textRlkXIni">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textRlkAlpha">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textRlkSigma">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>189</x>
         <y>2</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>I</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>1</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Alpha</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>181</x>
         <y>64</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Y Ini</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Sigma</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textRlkYIni">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textRlkMu">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>6</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>62</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Mu</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
